lib/suspenders/adapters/heroku.rb

```diff
@@ -34,6 +33,46 @@ def set_heroku_rails_secrets
         end
       end
 
+      def provide_review_apps_setup_script
+        app_builder.template(
+          "bin_setup_review_app.erb",
+          "bin/setup_review_app",
+          force: true,
+        )
+        app_builder.run "chmod a+x bin/setup_review_app"
+      end
+
+      def create_heroku_pipelines_config_file
+        template "app.json.erb", "app.json"
+      end
+
+      def create_heroku_pipeline
+        heroku_app_name = app_name.dasherize
+
+        pipelines_plugin = `heroku plugins | grep pipelines`
+        if pipelines_plugin.empty?
+          puts 'You need heroku pipelines plugin. Run: heroku plugins:install heroku-pipelines'
+          exit 1
```

Comment:
> This feels a bit too harsh. I think that If the pipelines plugin is not installed, we should just setup the applications as normal. And not completely stop.

This is inline with our goal of making `suspenders` command fail when any sub-action fails (https://github.com/thoughtbot/suspenders/issues/493). I'd rather know that something went wrong by not having an application generated, rather than having to scroll up the logs and find out what might have failed.